run-tests.sh

```diff
@@ -14,14 +50,41 @@ do
   full_second_level_dir=$(echo "$dir" | cut -d/ -f1-2)
 
   # Only run if change happened at second level, since first level-changes don't require the tests to be re-run.
+  # If change happened at first level, $second_level_dir will be empty.
   if [ ! -z "$second_level_dir" -a "$second_level_dir" != " " ]; then
     if [[ ! " ${tested_dirs[*]} " =~ " ${full_second_level_dir} " ]]; then
+      echo "Executing tests for $full_second_level_dir"
+
       tested_dirs+=($full_second_level_dir)
       if [ -e $full_second_level_dir/requirements.txt ]; then
-        python -m pip install -r $full_second_level_dir/requirements.txt --extra-index-url https://download.pytorch.org/whl/cpu
+        $1 -m pip -q install -r $full_second_level_dir/requirements.txt --extra-index-url https://download.pytorch.org/whl/cpu --no-warn-script-location
```

Comment:
Can you assign `$1` to a variable with a clearer name so this is a bit easier to follow?